otj-kafka/src/main/java/com/opentable/kafka/builders/KafkaProducerBuilder.java

```diff
@@ -93,10 +94,16 @@ public KafkaProducerBuilder(Map<String, Object> prop, EnvironmentProvider enviro
 
 
     public KafkaProducerBuilder<K, V> withSamplingRatePer10Seconds(int rate) {
-        kafkaBaseBuilder.withSamplingRatePer10Seconds(rate);
+        kafkaBaseBuilder.withBucketedSamplingRate(rate, LoggingInterceptorConfig.DEFAULT_BUCKET_DENOMINATOR);
         return this;
     }
 
+    public KafkaProducerBuilder<K, V> withSamplingRatePerNSeconds(int rate, int nSeconds) {
```

Comment:
I would recommend adding javadoc for all of the sampling configuration methods, but especially the new one.